Firebase/Storage/FIRStorageMetadata.m

```diff
@@ -180,6 +180,10 @@ - (NSDictionary *)dictionaryRepresentation {
   if (_path) {
     metadataDictionary[kFIRStorageMetadataName] = _path;
   }
+  
```

Comment:
travis is still complaining about one line in FIRStorageMetadata.m with trailing spaces.

You'll see the file and line number from the command line if you do git grep -n ' $'